### DISCUSSION TITLE
Thoughts on the map section.

### DISCUSSION THREAD

**Discussion by Kedyn** [Kedyn/fusliez-notes#138] [Ideas]
As of right now, on the beta version the characters on the map do not remain on the position they were set. I am aware of this issue, I also know that is more of a problem on browsers such as chrome. I did made it so that the map remains on the left of the screen as seen [here](https://kedyn.github.io/fusliez-notes/next/). But the characters still move, even with this change, due to the scaling of the image. I was thinking of switching from svg to canvas. This might also open up other features such as zooming. I also know there are other options. Any ideas on this are welcome.

**Comment by francistse23** [comment]
*still seeing the map on the right side from your link

Not too familiar with canvas myself. Are we sure the positioning would get preserved after switching to canvas? I'm open to experimenting with different implementations until we find something that works

**Comment by Kedyn** [comment]
Here is a preview of what I see:

![image](https://user-images.githubusercontent.com/11042906/102559802-9637cf00-4085-11eb-9e11-5ba428921cf9.png)
![image](https://user-images.githubusercontent.com/11042906/102559819-a059cd80-4085-11eb-94b1-482bae2acdea.png)

We could use something like [react-svg-pan-zoom](https://www.npmjs.com/package/react-svg-pan-zoom), this would not solve the problem. It would only add the zoomable feature.

As for the canvas idea, it would be like creating a mini game. It would allow us to have control over the whole thing, but it would also be some work.

**Comment by Kedyn** [comment]
For anyone interested, I have moved on with my canvas idea. The changes can be viewed on the refactoring branch and also: https://kedyn.github.io/fusliez-notes/next/

This will not only fix #149 , but also allow us to add many of the things people asked, such as being able to draw on the map.
